pom.xml

```diff
@@ -172,7 +172,7 @@
   <distributionManagement>
     <repository>
       <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases</url>
+      <url>https://repo.jenkins-ci.org/releases/</url>
```

Comment:
It seems the valid url is `https://repo.jenkins.io/releases/` but I thought the idea was to move it to `repo.jenkins.io`.

Forget my comment.